src/systemd/cockpit-wsinstance-http.service.in

```diff
@@ -2,11 +2,11 @@
 Description=Cockpit Web Service http instance
 Documentation=man:cockpit-ws(8)
 BindsTo=cockpit.service
-Requires=cockpit-session.socket cockpit-session-socket-user.service
-After=cockpit-session.socket cockpit-session-socket-user.service
+Requires=cockpit-session.socket
+After=cockpit-session.socket
 
 [Service]
 ExecStart=@libexecdir@/cockpit-ws --no-tls --port=0
-User=cockpit-wsinstance-socket
-Group=cockpit-wsinstance-socket
-SupplementaryGroups=cockpit-session-socket
+DynamicUser=true
+User=cockpit-ws-service-http
```

Comment:
Did you do this on purpose, or is it a leftover? The commit message says that you let systemd pick the user names, and it should be `cockpit-wsinstance-http`. That makes sense to me, and makes our current zoo of units and users quite a bit nicer to understand. "cockpit-ws-service-http" feels rather arbitrary to me. This name also isn't referenced anywhere, so IMHO it'd be fine to just stick with the default here?

Reply:
This looks like an accidental inclusion, indeed.  Let's drop that.

Reply:
Thanks for confirming. Dropped.